subsys/logging/Kconfig.misc

```diff
@@ -56,6 +56,13 @@ config LOG2_FMT_SECTION
 	  removing strings from final binary and should be used for dictionary
 	  logging.
 
+config LOG2_USE_TAGGED_ARGUMENTS
+	bool "Using tagged arguments for packaging"
+	select CBPRINTF_PACKAGE_SUPPORT_TAGGED_ARGUMENTS
+	select LOG2_ALWAYS_RUNTIME
```

Comment:
Is there a plan to support tagging with static packaging?

Reply:
Yes. However, C++ and Cortex-M (mps2_an385) is not working correctly right now with static packaging. For some reason, this combination results in an additional empty string argument being added to the list before any arguments (and that the tagged argument code is not performed on the empty string). This does not happen if only doing C on mps2_an385, or on other architectures, or using runtime packaging. I haven't been able to figure out why yet. Will update once I find out why.